tasks/shared/fieldstatus/fieldStatusTracker_test.go

```diff
@@ -1,17 +1,19 @@
-package tasks
+package fieldstatus_test
 
 import (
 	"fmt"
 	"testing"
 
+	"github.com/realvnc-labs/tacoscript/tasks/shared/fieldstatus"
+	"github.com/realvnc-labs/tacoscript/tasks/shared/fieldstatus/statusbuilder"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 type TestTaskWithCombinedNameMapperAndChangeTracker struct {
-	Field1 string `taco:"field_1"`
-	Field2 string `taco:"field_2"`
-	Field3 string `taco:"field_3"`
+	Field1 string `taco:"field_1,true"`
```

Comment:
I don't think this `,true` syntax is very readable. Why not `,tracked` so it's more descriptive, similar to how json package has `,omitempty`.

Reply:
i much prefer this :) done.